src/emulator/storage/rules/utils.ts

```diff
@@ -50,6 +50,25 @@ export function getRulesValidator(rulesetProvider: RulesetProvider): RulesValida
   };
 }
 
+/**
+ * Returns a validator that only checks for admin credentials.
```

Comment:
Would be helpful to say what the difference is between `getAdminOnlyRulesValidator` and `getAdminCredentialValidator`, I think.